packages/edit-site/src/components/editor/global-styles-provider.js

```diff
@@ -140,7 +200,7 @@ export default function GlobalStylesProvider( {
 				( value ) => value?.settings || {}
 			),
 		} );
-	}, [ contexts, mergedStyles ] );
+	}, [ mergedStyles ] );
```

Comment:
Should we also include contexts here? There is no expectation that it changes, but it is a value coming from a store so technically the contexts may change. I guess a case where it may happen is if a new block is installed from the block directory.

Reply:
nice catch! fixed https://github.com/WordPress/gutenberg/pull/27099/commits/a120e96bb492885a7edcf5aa5550a409bbc25619